src/style.css

```diff
@@ -0,0 +1,23 @@
+body {
+  margin: 0;
+  background-color: #000;
+}
+
+.box {
+  background-color: #ddd;
+  width: 350px;
+  margin: 20px 0 40px 20px;
+}
+
+.box:last-child {
```

Comment:
Try to avoid increasing the specificity of the selector. One class is enough to identify the elements on the page. No need to write something like .block.colored-block. Use .block and .block--colored instead, and apply each of them to your element

Reply:
All changed by your recommendations!